src/bindings/pygaia/fusion.py

```diff
@@ -71,6 +71,11 @@ def mergeChunk(points, outputFilename, transfoFile, start = 0, end = 1000000, se
             exclude = [ exclude ]
         cmd += [ '--exclude=' + ','.join(exclude) ]
 
+        # decide if we want the loader to fail if a pattern to exclude is not
+        # found in the point layout
+        if not failOnUnmatched:
+            cmd += [ '--passUnmatched' ]
```

Comment:
out of interest, why do you invert the meaning of the parameter here? Is there a reason that it's better to use `pass` in gaiamerge?

Reply:
I inverted the meaning to preserve the default behavior (fail on unmatched) on `gaimerge` when no flags are specified just to prevent interference with code expecting the original behavior.

Reply:
I wonder if we could improve the name of the argument, then.
noFailOnUnmatched or dontFailOnUnmatched sound better to me.